app/styles/common.scss

```diff
@@ -49,24 +64,33 @@ a.disabled {
   cursor: default;
 }
 
-.question-popover{
+.question-popover {
   cursor: pointer;
   margin-left: 8px;
   color: #337ab7;
-  &:hover{
+  &:hover {
     color: #23527c;
   }
 }
 
+.warning-popover {
+  cursor: pointer;
+  margin-left: 8px;
+  color: $brand-warning;
+  &:hover {
+    color: $brand-warning;
+  }
+}
+
```

Comment:
Sorry for all the other changes in this file. Prettier formatted it on commit. These are the only real changes in this file.